### DISCUSSION TITLE
Impossible to download france.tv

### DISCUSSION THREAD

**Discussion by dverdier** [aclap-dev/video-downloadhelper#1174] [Help and Q&A]
### Your question

Hello and thank you in advance for your reply. 
I use the france.tv platform and I'm sorry to say that video downloadhelper is unable to download any video on this site. 

I'm using the latest version (updated yesterday) and I work on a Mac. 

Thank you for your help.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"ca39187217cd4f26"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "valid_license_message_has_been_displayed": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 5,
    "successfull_dl": 21
}

**Comment by jcfrog** [comment]
Hi,
I managed to download this one: https://www.france.tv/documentaires/documentaires-sport/6348575-a-corps-perdus.html
Low resolution but plays fine. Can you try?

**Reply by dverdier** [reply]
Sorry but... no, it doesn't work (it fails with low or high resolution)
Could you try with this one ? 
https://www.france.tv/films/films-suspense/5755230-plein-soleil.html

**Reply by jcfrog** [reply]
it works for me, but always low definition

**Reply by jcfrog** [reply]
<img width="335" alt="image" src="https://github.com/user-attachments/assets/74d45957-a64c-47fa-aa5e-787819260b88">

**Reply by dverdier** [reply]
It's really strange. I have the same suggestion but the download stops after a few seconds. 
Another strange thing: it offers you 52 minutes when the film is 113 minutes long. Can you please check this?

**Comment by Mattmjs** [comment]
And does it work on other sites or are there problems ?
